app/code/Magento/InventoryCatalog/Test/Integration/GetDefaultSourceItemBySkuTest.php

```diff
@@ -39,9 +39,5 @@ public function testExecute()
             $this->getDefaultSourceItemBySku->execute('SKU-2'),
             'Unable to find default source_item for a product assigned to Default source and others'
         );
-        self::assertNull(
-            $this->getDefaultSourceItemBySku->execute('SKU-3'),
-            'Returned a wrong default source_item for a product assigned elsewhere'
-        );
```

Comment:
hey @phoenix128 , why did you remove this assertion?

Reply:
See my new commit, I just forgot to create the new method with different fixtures